packages/e2e-tests/src/tests/connection.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect, jest, test } from '@jest/globals';
+import { TinnyEnvironment, LIT_TESTNET } from '@lit-protocol/tinny';
+
+try {
+  jest.setTimeout(60000);
+} catch (e) {
+  // ... continue execution
+}
+
+const NETWORKS: string[] = ['manzano', 'datil-dev', 'cayenne'];
+
+describe('Connections', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'warn').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    // TODO: can be removed once v7 is merged with wasm refactors
+    delete globalThis.wasmExports;
```

Comment:
This should be happening in `disconnect()` if we await it (it calls `unloadModules`)